js/main.js

```diff
@@ -365,6 +365,13 @@
       parseRepos(ghApi, [$(this).data().repoPath]);
     });
 
+    $('#approved-prs').on('click', 'td', function() {
+      var link = $(this).find('a');
+      if (link.length === 1 && link.attr('href')) {
+        window.open(link.attr('href'));
```

Comment:
I like `link.click()` better but it doesn't work.  I added the `_blank` parameter.